### PR TITLE
Feat/brdf

### PR DESCRIPTION
**Pull request by maximlamare**
Add BRDF page to custom scripts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,7 +230,8 @@ Dedicated to supplying data for [Copernicus services](http://www.esa.int/Our_Act
  - [MixLAI](sentinel-2/mixlai) - Mix Leaf Area Index
  - [Detection of evapotranspiration levels](sentinel-2/evapotranspiration_levels)
  - [Aesthetic Neon](sentinel-2/aesthetic-neon) - Aesthetic visualization for urban and dry (desert) areas
- - [Total Ozone Column over Antarctica snow](sentinel-2/ozone_column_over_snow)
+ - [Total Ozone Column over Antarctica snow](sentinel-2/brdf)
+ - [BRDF normalisation](sentinel-2/ozone_column_over_snow)
```
Comment (jonasViehweger) [inline README.md:234]:
```suggestion
 - [Total Ozone Column over Antarctica snow](sentinel-2/ozone_column_over_snow)
 - [BRDF normalisation](sentinel-2/brdf)
```

**Review by jonasViehweger — COMMENTED** [review]
Looks good, only a small mistake and I would change the script a bit

**Comment by jonasViehweger** [comment]
From my side everything looks good. I did some changes, please check them out. After that it should be ready to merge

File: sentinel-2/brdf/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+The Evalscript proposed here is the "semi-empirical BRDF normalisation c-factor correction approach proposed by Luch et al. 2000 [1]. The formula is detailed in eq. 37 (using eq. 38 & 39). To avoid lookup tables and other complex modelling approaches, we use the fixed BRDF spectral model parameters derived from "the global year of highest quality snow-free MODIS BRDF product" conveniently defined for Sentinel-2 in Roy et al. 2017 [2] that were derived from the Landsat ones in Roy et al. 2016 [3].
+
+The approach seems to perform generally quite well for most surfaces (except snow and ice) and high sun angles. Nevertherless, it relies on a certain number of approximations and may not always be suitable.
+
+## Usage
+
+BRDF corrections are available for Sentinel 2 bands 2, 3, 4, 8, 11 and 12. In the beginning of the script there is the option to define the bands which the correction should be applied to. It is also possible to set a luminance value in case RGB images are supposed to be generated.
```
Comment (maximlamare) [inline sentinel-2/brdf/README.md:23]: I would add information about the luminance value:

" The luminance value is a multiplication factor designed to brighten the image for a better visualisation. By default, afactor of 2.5 is applied which works for most surface types. For darker surfaces, such as forests, a higher value may be required."

**Pull request merged**